config/locales/en.yml

```diff
@@ -267,3 +267,7 @@ en:
       table_headings:
         <<: *TABLE_HEADINGS
 
+  pagination:
```

Comment:
Maybe we could add an issue about looking through the en.yml and doing some spring cleaning / reorg as they have on Apply 🤔

Reply:
Added an issue for doing this: https://github.com/ministryofjustice/laa-review-criminal-legal-aid/issues/107